README.md

```diff
@@ -3,3 +3,25 @@
 This documentation will walk you through the steps of building and managing an Ubuntu Core device.
 
 You can find it published on [docs.ubuntu.com/core](http://docs.ubuntu.com/core).
+
+# Building
+
+To build the documentation you need to install documentation-builder:
+
+    $ snap install documentation-builder
```

Comment:
It should be the right format as the Markdown syntax is just to indent with at least 4 spaces to get a codeblock.